Sources/MongoSwift/BSON/BsonValue.swift

```diff
@@ -205,7 +205,7 @@ public struct Binary: BsonValue, Equatable, Codable {
     /// Initializes a `Binary` instance from a `Data` object and a `UInt32` subtype.
     internal init(data: Data, subtype: UInt32) {
         self.data = data
-        self.subtype = BsonSubtype(rawValue: Int(subtype))!
+        self.subtype = Subtype(rawValue: Int(subtype))!
```

Comment:
This may be outside the scope of this PR, but is it possible to validate the sub-type as an unsigned 8-bit integer? See https://jira.mongodb.org/browse/PHPC-593.

Also, does constructing a Subtype enum here cause problems if users use a value other than one of the pre-defined constants? We should allow all values within the range of 0-255 (i.e. 0x00-0xFF).